Python requirements & Devcontainers (#69)
OpenDataServices/os4d: README.md

This section describes how to:

> - [Use dev containers or GitHub CodeSpaces for development](#use-dev-containers-or-github-codespaces-for-development)

Reviewer: Capitalisation nitpick :-)

```suggestion
- [Use dev containers or GitHub Codespaces for development](#use-dev-containers-or-github-codespaces-for-development)
```

- [Set up a local development environment](#set-up-a-local-development-environment)
- [Build the documentation](#build-the-documentation) on your local machine.
- [Deploy changes](#deploy-changes)
- [Update requirements](#update-requirements)

### Use dev containers or GitHub CodeSpaces for development

> If your IDE supports dev containers, you can open the project in a dev container.

Reviewer: Link to dev containers documentation (not sure if there's a better non-editor-specific source) and add example instructions for VS Code.
 
```suggestion
If your editor supports [dev containers](https://containers.dev/), you can open the project in a dev container. For example, if you're using VS Code, follow the [installation instructions](https://code.visualstudio.com/docs/devcontainers/containers#_installation) and the quick start guide to [open a folder in a container](https://code.visualstudio.com/docs/devcontainers/containers#_quick-start-open-an-existing-folder-in-a-container). 
```

> You can also open the project in GitHub CodeSpaces.

Reviewer: ```suggestion
You can also open the project in GitHub Codespaces.
```

> Once open, open a new terminal and see the "Build the documentation" section below for the commands you need to run.

Reviewer: Avoid repetition of 'open', add link.

```suggestion
Once connected to the container, open a new terminal and run the commands in [Build the documentation](#build-the-documentation).
```

> When you have finished, make sure you delete you workspace on https://github.com/codespaces

Reviewer: ```suggestion
When you have finished, make sure to delete your codespace on https://github.com/codespaces.
```
